app/ui-react/packages/ui/src/Connection/ConnectionsGrid.tsx

```diff
@@ -1,12 +1,8 @@
-import { CardGrid } from 'patternfly-react';
+import { Gallery } from '@patternfly/react-core';
 import * as React from 'react';
 
 export class ConnectionsGrid extends React.Component {
   public render() {
-    return (
-      <CardGrid fluid={true} matchHeight={true}>
-        <CardGrid.Row>{this.props.children}</CardGrid.Row>
-      </CardGrid>
-    );
+    return <Gallery gutter={'sm'}>{this.props.children}</Gallery>;
```

Comment:
This creates the empty state as a grid item, too. Can we only include the gallery layout if there are connections?

<img width="1419" alt="Screen Shot 2019-07-30 at 5 09 07 PM" src="https://user-images.githubusercontent.com/35148959/62169440-d8743b00-b2ed-11e9-8567-65754d5d6a16.png">

Reply:
Also looks like you're missing a global variable (`--pf-l-gallery--m-gutter--GridGap`) that the gallery uses to define space responsively. That var comes from the core base CSS. Is it possible you have a newer version of the `Gallery` component than the base CSS?

<img width="658" alt="Screen Shot 2019-07-30 at 5 15 43 PM" src="https://user-images.githubusercontent.com/35148959/62169522-0e192400-b2ee-11e9-9531-c1614428d340.png">